diff --git a/src/nvc_info.c b/src/nvc_info.c
index 24e432a..1e091fc 100644
--- a/src/nvc_info.c
+++ b/src/nvc_info.c
@@ -211,8 +211,6 @@ find_library_paths(struct error *err, struct dxcore_context *dxcore, struct nvc_
         if (path_resolve_full(err, path, root, ldcache) < 0)
                 return (-1);
         ldcache_init(&ld, err, path);
-        if (ldcache_open(&ld) < 0)
-                return (-1);
 
         info->nlibs = size;
         info->libs = array_new(err, size);
@@ -232,8 +230,6 @@ find_library_paths(struct error *err, struct dxcore_context *dxcore, struct nvc_
         rv = 0;
 
  fail:
-        if (ldcache_close(&ld) < 0)
-                return (-1);
         return (rv);
 }
